festung/_private.py

```diff
@@ -77,6 +77,8 @@ def qs_to_headers(url):
 
 def cast(obj):
     """Cast most python types to valid types to be serialized in JSON and passed to SQLite"""
+    if isinstance(obj, bool):
```

Comment:
Maybe we should add a comment, why we need to check boolean first, because this is not immediately obvious.